### PR TITLE
[Docs] adds `globalVariable` function

### LINKED ISSUE #2702: Document `globalVariable` function
Create documentation for the `globalVariable` function. See https://github.com/ClickHouse/clickhouse-docs/issues/1833 for details on how to complete this issue.

Child issue of https://github.com/ClickHouse/clickhouse-docs/issues/1833.

### PR DESCRIPTION
**Pull request by Blargian**
Closes [#2702](https://github.com/ClickHouse/clickhouse-docs/issues/2702).

Adds function documentation for missing `globalVariable` .

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Review by pamarcos — APPROVED** [review]
Good work :rocket:
Thanks a lot for your contribution, Shaun! :bow:

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 56fe7206d0bba34af44275326863581bab43c599 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/70607/56fe7206d0bba34af44275326863581bab43c599/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/70607/56fe7206d0bba34af44275326863581bab43c599/style_check.html">❌ failure</a></td></tr>
<tbody>
</table>

File: docs/en/sql-reference/functions/other-functions.md [hunk 1]
```diff
@@ -4390,3 +4390,37 @@ Result:
 1. │ ['{ArraySizes}','{ArrayElements, TupleElement(keys), Regular}','{ArrayElements, TupleElement(values), Regular}'] │
    └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+
+## globalVariable
+
+Takes constant string argument and returns the value of global variable with that name. It is intended for compatibility with MySQL.
```
Comment (rschu1ze) [inline docs/en/sql-reference/functions/other-functions.md:4396]: Could add something like "This function is not useful for normal operation of ClickHouse."

Comment (pamarcos) [reply docs/en/sql-reference/functions/other-functions.md:4396]: Created a new PR: https://github.com/ClickHouse/ClickHouse/pull/70938

**Pull request merged**